### DISCUSSION TITLE
How to add created event in google calendar ?

### DISCUSSION THREAD

**Discussion by Dharmraj12** [ical4j/ical4j#776] [Q&A]
**Describe the bug**
I am creating meeting using ical4j . its creating calendar and ics file . but its not added in google calendar  automatic. its showing add to calendar  option in 
https://github.com/ical4j/ical4j/wiki/Examples#Creating_a_meeting_of_four_hour_duration
**To Reproduce**

**Expected behavior**
that meeting should be added in google calendar 

**Screenshots**
![Not added in google calendar](https://user-images.githubusercontent.com/29793162/82634546-91a85680-9c1b-11ea-8b02-fb28ab598f63.png)

**Environment (please complete the following information):**
 - OS: [e.g. Windows, Linux, Android]
 - Java Version [e.g. 7, 8, 9]

**Additional context**

**Comment by benfortuna** [comment]
You might need to add a `METHOD` to the calendar object properties. e.g. `METHOD:PUBLISH`

**Comment by Dharmraj12** [comment]
@benfortuna  I have tried METHOD.REQUEST, METHOD.PUBLISH but still its not working

**Comment by Dharmraj12** [comment]
for( String  email : to ) {
		 Attendee dev1 = new Attendee(URI.create("mailto:" + email.trim()));
		 dev1.getParameters().add(Role.REQ_PARTICIPANT);
		 dev1.getParameters().add(Rsvp.TRUE);
		 dev1.getParameters().add(PartStat.NEEDS_ACTION);
		 meeting.getProperties().add(dev1);
	 }
	 	meeting.getProperties().add(new Description(mailModel.getDescription()));
		meeting.getProperties().add(new Sequence(0));
		meeting.getProperties().add(Transp.OPAQUE);
		meeting.getProperties().add(new Location(mailModel.getLocation()));
    	 meeting.getProperties().add(Status.VEVENT_CONFIRMED);
    	 meeting.getProperties().add(Method.PUBLISH);

**Comment by benfortuna** [comment]
Hi, just to confirm is the issue that the event isn't added to Google Calendar automatically? I don't think there is a way to do this, as just like any other CUA you must "accept" the event to add to your calendar.
